models/double_entry_transactions/int_quickbooks__bill_payment_double_entry.sql

```diff
@@ -45,7 +58,9 @@ bill_payment_join as (
             order by bill_payments.source_relation, bill_payments.transaction_date) - 1 as index,
         bill_payments.transaction_date,
         bill_payments.total_amount as amount,
-        (bill_payments.total_amount * coalesce(bill_payments.exchange_rate, 1)) as converted_amount,
+        case when bill_payment_lines.journal_entry_id is not null 
+             then ((-1) * bill_payment_lines.amount)* (coalesce(coalesce(bills.exchange_rate, bill_payments.exchange_rate), 1))
```

Comment:
Small spacing suggestion
```suggestion
             then ((-1) * bill_payment_lines.amount) * (coalesce(coalesce(bills.exchange_rate, bill_payments.exchange_rate), 1))
```

Reply:
Updated